Client/Ecosia/Extensions/AppDelegate.Ecosia.swift

```diff
@@ -18,5 +18,31 @@ extension AppDelegate {
             }
         }
     }
+}
 
+extension AppDelegate {
+    
+    func sendAppAndDeviceInfoToMobileMeasurementPlatformService() {
+        Task {
+            do {
+                let appDeviceInfo = AppDeviceInfo(identifier: UUID().uuidString,
```

Comment:
I guess we need to store this id somewhere, don't we?

Reply:
💯 . Added to make sure we complete the e2e testing.